packages/amplify_authenticator/lib/src/mixins/authenticator_phone_field.dart

```diff
@@ -158,9 +157,7 @@ mixin AuthenticatorPhoneFieldMixin<FieldType,
     );
 
     if (selectedCountry != null) {
-      setState(() {
-        _selectedCountry = selectedCountry;
-      });
+      state.country = selectedCountry;
```

Comment:
Seems like `setState` would still be needed, no?

Reply:
The country is now stored in AuthenticatorState and the setter for country calls `notifyListeners()`.